sdk/servicebus/service-bus/src/session/messageSession.ts

```diff
@@ -470,7 +470,7 @@ export class MessageSession extends LinkEntity {
   /**
    * Closes the underlying AMQP receiver link.
    */
-  async close(): Promise<void> {
+  async close(isSessionExpired?: boolean): Promise<void> {
```

Comment:
What do you think of renaming this to `closeDueToSessionExpiry`?

It makes the intention of the parameter more clear and avoid future developers in assuming that they need to think about session expiry before calling close.

Reply:
Oh, good point! Updated and made this internal to messageSession.
Please see latest commit.